recipe/migrations/cuda128.yaml

```diff
@@ -0,0 +1,61 @@
+migrator_ts: 1738229377
+__migrator:
+  kind:
+    version
+  migration_number:
+    1
+  build_number:
+    1
+  paused: false
+  override_cbc_keys:
+    - cuda_compiler_stub
+  operation: key_add
+  check_solvable: false
+  primary_key: cuda_compiler_version
+  ordering:
+    cuda_compiler:
+      - None
+      - nvcc
+      - cuda-nvcc
+    cuda_compiler_version:
+      - None
+      - 11.8
+      - 12.4
+      - 12.6
+      - 12.8
+  commit_message: |
+    Upgrade to CUDA 12.8
+    
+    With CUDA 12.8, the following new architectures are added `sm_100`, `sm_101` and `sm_120`.
+    To build for these architectures, maintainers will need to add these to list of architectures
+    that their package builds for.
+    
+    ref: https://docs.nvidia.com/cuda/cuda-toolkit-release-notes/index.html#new-features
```

Comment:
@carterbox could you please take a look at this messaging and suggest any improvements that you have in mind?